### DISCUSSION TITLE
After first Uninstall Is Impossibile Reinstall NvChad Without Bugs

### DISCUSSION THREAD

**Discussion by clemenzi** [NvChad/NvChad#1187] [Q&A] [answered]
Hello! I I can't reinstall NvChad after deleting it a first time. How can I reinstall NvChad without problems?

**Comment by siduck** [comment, marked as answer]
what problems do you get

**Reply by clemenzi** [reply]
It installs without problems but the graphics seem "corrupted" (?)
Look at this screen:

https://i.imgur.com/rwWDNWT.png

**Reply by siduck** [reply]
@Thebigbot0000 did you had packer installed before?

**Reply by clemenzi** [reply]
@siduck Yes, only once did I uninstall it. Now I was trying to reinstall it but it gives me this graphic bug

**Reply by siduck** [reply]
@Thebigbot0000 do a packersync

**Reply by clemenzi** [reply]
@siduck Ok, thanks I managed to solve with your instructions. Have a nice day
